src/tui.rs

```diff
@@ -109,7 +110,23 @@ pub fn render_tui(key:String, proj_name: &str){
                                     .expect("there is always a selected template"),
                             )
                             .unwrap();
-                    command::git_clone(proj_name, selected_template.url.to_string(),&mut terminal);
+
+                    disable_raw_mode().unwrap();
+                    execute!(
+                            terminal.backend_mut(),
+                            LeaveAlternateScreen,
+                        ).unwrap();
+                    println!("\nCloning {}..\n", proj_name);
+                    let output = command::git_clone(proj_name, selected_template.url.to_string());
+                    terminal.show_cursor().unwrap();
+                    if output.status.success() {
+                            println!("\nCloned {} successfully\n", &proj_name);
+                        } else {
+                            println!(
+                                "\nError Encountered while cloning, {:?}",
```

Comment:
Can you print the error string by unwrapping output instead of printing using the debug flag?

Reply:
Opened issue & Fixed #13

Reply:
Resolved!